content/api/servo-assembly/GPIO.md

```diff
@@ -37,9 +37,9 @@ from robot import PinMode
 r.servo_board.gpios[3].mode = PinMode.INPUT_PULLUP
 ```
 
-## Reading a pin value
+## Reading a pin state
 
-To read the value of a pin, simply call `read()`. This will return either `PinValue.HIGH` or `PinValue.LOW`, which have to be imported to use.
+To read the state of a pin, simply call `read()`, which will return either `PinValue.HIGH` or `PinValue.LOW`. Like `PinMode`, the `PinValue` enum must be imported.
```

Comment:
I think we should keep (or add) the phrasing "imported to be used". Otherwise it could be read that calling `.read()` without the import would do something undefined.

Reply:
Agreed, done.